core/src/test/java/io/grpc/internal/ServiceConfigInterceptorTest.java

```diff
@@ -351,9 +363,9 @@ public void handleUpdate_matchNames() {
     assertThat(interceptor.serviceMethodMap.get())
         .containsExactly(
             methodDescriptor.getFullMethodName(),
-            new MethodInfo(methodConfig));
-    assertThat(interceptor.serviceMap.get())
-        .containsExactly("service2", new MethodInfo(methodConfig));
+            new MethodInfo(methodConfig, false, 1));
```

Comment:
The `MethodInfo.retryPolicy` does not have `maxRetryAttempts` equal to 1. This makes these lines kind of confusing, especially since `MethodInfo.equals` ignores the `retryPolicy` altogether.

Should `MethodInfo.equals` compare retry policy as well?

Either way, it might be clearer to note that 1 is ignored since retryEnabled=false